src/components/upnextdialog/upnextdialog.js

```diff
@@ -256,7 +256,7 @@ import 'flexStyles';
             const runtimeTicks = playbackManager.duration(options.player);
 
             if (runtimeTicks) {
-                const timeRemainingTicks = runtimeTicks - playbackManager.currentTime(options.player);
+                const timeRemainingTicks = runtimeTicks - playbackManager.currentTime(options.player) * 10000;
```

Comment:
Converting playbackManager to operate in ms (preferred solution, imo) would allow us to completely drop this function.